scripts/audit.sh

```diff
@@ -1,18 +1,23 @@
 #!/usr/bin/env bash
 set -uo pipefail
 
-PROJECT_ROOT="$(cd "$(dirname "$0")" && pwd)/../"
-
 CI="${CI:-false}"
 
-source "$PROJECT_ROOT/scripts/include.sh"
+source "./scripts/include.sh"
+
+LICENSE_LIST="$(jq -c -r '.[]' < ./licenses.json)"
 
-ALLOWED_LICENSES=$(echo "$(cat $PROJECT_ROOT/licenses.json)" | jq -c '.[]')
-if [ "$ALLOWED_LICENSES" == "" ]; then
+if [ "$LICENSE_LIST" == "" ]; then
     exitonfail 1 "License list import"
 fi
 
-npx license-checker --onlyAllow "$(echo $ALLOWED_LICENSES | sed -E "s/\" /;/g" | sed -E "s/\"//g")"
+ALLOWED_LICENSES=""
+IFS=$'\n'
+for LICENSE in $LICENSE_LIST; do
+    ALLOWED_LICENSES="${ALLOWED_LICENSES}${LICENSE};"
```

Comment:
NIT: Maybe we can go for purely concat approach with `+=` there? :)

In this case it'd be:
`ALLOWED_LICENSES+="${LICENSE};"`